### ISSUE TITLE
gtest-adapter.refreshAfterBuild doesn't work

### ISSUE THREAD

**Issue by ftrofin** [DavidSchuldenfrei/gtest-adapter#18]
Although I have `gtest-adapter.refreshAfterBuild` set to true (default), after I build, the test panel is not automatically refreshed. I am using "Run Build Task" (Shift + Cmd +B) that invokes the default build task. If I click the "refresh" button in the test panel then I get the list of tests populated in the panel.

Environment:
MacOS 10.13.6
VS Code Version 1.28.1 (1.28.1)
GoogleTest Adapter version 1.2.4
Latest gtest version (1.8.1)

**Comment by ftrofin** [comment]
Hi David,
The name is "Build unittester Debug" and here is how the entry looks in tasks.json:
```        {
            "label": "Build unittester Debug",
            "type": "shell",
            "command": "xcodebuild -project ${workspaceRoot}/makeTests.mp.xcodeproj -scheme unittester.Debug build |xcpretty",
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "problemMatcher": []
        },```

**Comment by DavidSchuldenfrei** [comment]
My code assumed the name of the task to be `build`. Serves me right for being lazy and not checking for the default task. I will try to have a fix out in a couple of days

**Comment by DavidSchuldenfrei** [comment]
This should be fixed with v 1.3.1
If you still have problems , please open a new issue.
